data/brands/tourism/hotel.json

```diff
@@ -217,6 +217,7 @@
       "locationSet": {"include": ["001"]},
```

Comment:
There's few enough countries served that we should add them here

Reply:
Makes sense to me, honestly.